tutorials/getting_started/crf_tagger.json

```diff
@@ -0,0 +1,62 @@
+{
+    "dataset_reader": {
+      "type": "conll2003",
+      "tag_label": "ner",
+      "token_indexers": {
+        "tokens": {
+          "type": "single_id",
+          "lowercase_tokens": true
+        },
+        "token_characters": {
+          "type": "characters"
+        }
+      }
+    },
+    "train_data_path": "/Users/joelg/data/ner/eng.train.json",
```

Comment:
Can we make the checked-in file here use dummy paths, instead of these?  And s3 links where appropriate, e.g., for the glove vectors.

Reply:
done, and I uploaded the 50d glove vectors to S3